packages/bitcore-wallet-service/src/lib/fiatrateservice.ts

```diff
@@ -168,6 +153,56 @@ export class FiatRateService {
   getRates(opts, cb) {
     $.shouldBeFunction(cb, 'Failed state: type error (cb not a function) at <getRates()>');
 
+    opts = opts || {};
+
+    const now = Date.now();
+    const ts = opts.ts ? opts.ts : now;
+    let fiatFiltered = [];
+    let rates = [];
+
+    if (opts.code) {
```

Comment:
is code really optional? what is the expected behavior is not given?

Reply:
Yes, it is optional, if you don't pass it as a parameter then you get the rates in all fiat supported. if you pass it as a parameter you only get the rate of the specified fiat (code)